binaries/pip/build_wheels.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+source ../../scripts/install_utils
+
+cleanup()
+{
+  rm -rf dist
+  rm -rf build
+  rm -rf *.egg-info
+}
+
+create_torchserve_wheel()
+{
+  cd ../../
+
+  python setup.py bdist_wheel --release --universal
+
+  cp dist/*.whl binaries/pip/output/
+
+  cleanup
+
+  cd -
+}
+
+create_model_archiver_wheel()
+{
+  cd ../../model-archiver
+
+  python setup.py bdist_wheel --release --universal
+
+  cp dist/*.whl ../binaries/pip/output/
+
+  cleanup
```

Comment:
To make it so that cleanup is always run, regardless of where you exit in the script you can use a `trap`, see https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_12_02.html

Reply:
Done.